teamworkApp/db/schema.rb

```diff
@@ -20,10 +20,43 @@
     t.integer "question"
   end
 
+  create_table "courses", force: :cascade do |t|
+    t.string "name"
+    t.integer "year"
+    t.integer "semester"
+    t.integer "section"
+    t.string "course_code"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "enrollments", force: :cascade do |t|
+    t.integer "student_id"
+    t.integer "course_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
   create_table "students", force: :cascade do |t|
-    t.integer "style"
     t.string "name"
-    t.integer "team"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "team_id"
+    t.string "username"
+  end
+
+  create_table "styles", force: :cascade do |t|
+    t.integer "student_id"
+    t.integer "communicator"
+    t.integer "collaborator"
+    t.integer "challenger"
+    t.integer "contributor"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "teams", force: :cascade do |t|
+    t.string "name"
```

Comment:
How does this connect to the data about which students are in the teams? I see there is a string `name` in the teams table, but an integer `team_id` in the students table. I could just be confused with the structure of databases and how these connect. I could also be missing another detail...

Reply:
So teams have many students, which means to represent teams without requiring a fixed size of team, we need the students to each have an attribute which identifies the team they are on. The team_id in students allows this to happen.

Reply:
Ok, so how does this connect back to the teams table? Like, how could you know the name of their team?